pkgs/development/tools/language-servers/vhdl_ls/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, stdenv
+, rustPlatform
+, fetchCrate
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "vhdl_ls";
```

```suggestion
  pname = "vhdl-ls";
```

Usually: `package-name_1_2_3`
Any reason not to?